internal/pkg/template/workload.go

```diff
@@ -735,7 +735,7 @@ func envControllerParameters(o WorkloadOpts) []string {
 		}
 	}
 	if o.WorkloadType == "Request-Driven Web Service" {
-		if o.Private {
+		if o.Private && o.AppRunnerVPCEndpoint == nil {
```

Comment:
Do we not have any unit tests for the template functions? I feel like it would be nice to have a unit test for `envControllerParameters`

Reply:
added!